nvdaHelper/vbufBackends/gecko_ia2/gecko_ia2.cpp

```diff
@@ -1262,11 +1262,13 @@ VBufStorage_fieldNode_t* GeckoVBufBackend_t::fillVBuf(
 			parentNode->isBlock=false;
 		}
 
-		if ((isInteractive || role == ROLE_SYSTEM_SEPARATOR) && parentNode->getLength() == 0) {
+		if ((isInteractive || role == ROLE_SYSTEM_SEPARATOR || name || description.has_value()) && parentNode->getLength() == 0) {

```

Comment:
Should `name` be changed to `name.has_value()` here?

Reply:
Not exactly, as `name` is a `BSTR`, not an `optional<wstring>`. I have checked that it's non-null and non-empty in my updated code though.